Task_TheGridProblem/index.css

```diff
@@ -1 +1,11 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */

```

Comment:
Consider specifying which elements the universal selector (*) should apply to for better specificity and performance.